### PR TITLE
session: migrate test-infra to testify for testBackupRestoreSuite

### PR DESCRIPTION
**Pull request by tisonkun**
Issue Number: ref #28328 

```release-note
None
```

### CODE REVIEW

File: session/backup_restore_test.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	if !*withTiKV {
+		t.Skip("only run BR SQL integration test with tikv store")
+	}
+
+	store, clean := createStorage(t)
+	defer clean()
+
+	cfg := config.GetGlobalConfig()
+	cfg.Store = "tikv"
+	//cfg.Path = pdAddr
```
[the remark this answers is not in the document]
Comment (tisonkun) [reply session/backup_restore_test.go]: Well. We still need to config this.

**Review by xhebox — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**